README.md

```diff
@@ -28,9 +26,6 @@ cd website
 The Kubernetes website uses the [Docsy Hugo theme](https://github.com/google/docsy#readme). Even if you plan to run the website in a container, we strongly recommend pulling in the submodule and other development dependencies by running the following:
 
 ```
-# install dependencies
-yarn
```

Comment:
Should this say `npm ci`?

Reply:
no it should just be npm. npm ci is a command...

Reply:
`git submodule update --init --recursive --depth 1` is a command too @inductor. These steps are about getting ready to run `make serve`

Reply:
@sftim I moved this line to https://github.com/kubernetes/website/pull/22995/files#diff-04c6e90faac2675aa89e2176d2eec7d8R53 because this is unnecessary for containerd solution.

Reply:
OK, makes sense.